src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -225,6 +225,65 @@ <h3 style="padding-left: 0.2rem;">JJ decision</h3>
     </mat-label>
     <div class="section-grid-text" style="padding-bottom: 2rem !important;">{{ jjDisputedCount?.comments }}</div>
   </div>
+  <!-- JJ Latest Change Plea -->
+  <section [formGroup]="form" *ngIf="type !== 'ticket' && jjDisputeInfo.hearingType === HearingType.CourtAppearance"
+    style="padding: 0 0.2rem;">
+    <br>
+    <div class="row">
+      <div class="col-md-4">
+        <span class="section-grid-header">Change Plea</span>
+      </div>
+      <div class="col-md-8">
+        <mat-select class="section-grid-text" formControlName="latestPlea" *ngIf="!isViewOnly && jjDisputedCount" 
+          style="width:75%; background:#fffee6" >
+          <mat-option [value]="LatestPlea.G">Guilty</mat-option>
+          <mat-option [value]="LatestPlea.N">Not guilty</mat-option>
+        </mat-select>
+        <div *ngIf="isViewOnly && jjDisputedCount">
+          <span class="section-grid-text" *ngIf="jjDisputedCount?.latestPlea === LatestPlea.G">Guilty</span>
+          <span class="section-grid-text" *ngIf="jjDisputedCount?.latestPlea === LatestPlea.N">Not guilty</span>
+        </div>
+      </div>
+    </div>
+    <div class="row mt-1">
+      <div class="col-md-4">
+        <span class="section-grid-header">Date/Time</span>
+      </div>
+      <div class="col-md-8">
+        <div *ngIf="!isViewOnly && jjDisputedCount">
+          <input type="datetime-local" formControlName="latestPleaUpdateTs" class="section-grid-text latest-plea-text" >
+        </div>
+        <div *ngIf="isViewOnly && jjDisputedCount && jjDisputedCount?.latestPleaUpdateTs">
+          <span class="section-grid-text">{{ jjDisputedCount?.latestPleaUpdateTs | date: "MM/dd/YYYY HH:mm" }}</span>
```

Comment:
Just confirming the date format is this, and not YYYY/MM/dd HH:mm (which is the format on other screens). But it's true that I don't think project has a standard.

Reply:
I have put this format similar to other input date formats